web/docs/data-model/operations/queries.md

```diff
@@ -188,7 +188,11 @@ For a detailed explanation of the Query definition API (i.e., arguments and retu
 
 ### Using Queries
 
-To call a Query in the client code, you can import it from `wasp/client/operations` and call it directly.
+#### Using Queries on the client
```

Comment:
I didn't update the API reference because:
 - It already points to this example of usage, which I think is clear enough
 - It's something they define themselves, not a core part of Wasp's API.
 - Documenting the exact type signature of the public API in the API reference would require shuffling stuff around, which is more work.
 
Let me know if you think I should update it regardless.